### PR TITLE
Wires Nano-UI-ification

### PR DESCRIPTION
**Pull request by davipatury**
:cl: davipatury
rscadd: Wires now uses Nano-UI.
/:cl:

image

### CODE REVIEW

**Comment by Markolie** [comment]
🙏

**Comment by Fox-McCloud** [comment]
Does this resolve the issue with vending machines not instantly updating their inventory when the contraband wire is pulsed?

**Comment by gt33** [comment]
Me likey.

**Comment by davipatury** [comment]
@Fox-McCloud Yes.

**Comment by Fox-McCloud** [comment]
Yay. Thanks.

**Comment by Fox-McCloud** [comment]
@Crazylemon64

File: code/datums/wires/wires.dm [hunk 1]
```diff
@@ -68,106 +68,99 @@ var/list/wireColours = list("red", "blue", "green", "black", "orange", "brown",
...
+					else
+						to_chat(L, "<span class='error'>You need a multitool!</span>")
+				if("attach")
+					if(IsAttached(colour))
+						var/obj/item/O = Detach(colour)
+						if(O)
+							L.put_in_hands(O)
+					else
+						if(istype(I, /obj/item/device/assembly/signaler))
+							L.drop_item()
```
Comment (Crazylemon64) [inline code/datums/wires/wires.dm]: NODROP check

**Comment by davipatury** [comment]
Fixed!

**Pull request merged**